webmails/roundcube/start.py

```diff
@@ -4,16 +4,61 @@
 import logging as log
 import sys
 from socrate import conf
+import subprocess
 
 log.basicConfig(stream=sys.stderr, level=os.environ.get("LOG_LEVEL", "WARNING"))
 
 os.environ["MAX_FILESIZE"] = str(int(int(os.environ.get("MESSAGE_SIZE_LIMIT"))*0.66/1048576))
 
+db_flavor=os.environ.get("ROUNDCUBE_DB_FLAVOR",os.environ.get("DB_FLAVOR","sqlite"))
+if db_flavor=="sqlite":
+    os.environ["DB_DSNW"]="sqlite:////data/roundcube.db"
+elif db_flavor=="mysql":
+    os.environ["DB_DSNW"]="mysql://%s:%s@%s/%s" % (
```

Comment:
Please use string.format, which is a more standard way of formatting strings nowadays.

Reply:
I'll change it accordingly (including the things you mentioned above)